x-pack/plugins/apm/dev_docs/testing.md

```diff
@@ -24,7 +24,8 @@ open target/coverage/jest/index.html
 | --trial   | Run tests with trial license                    |
 | --server  | Only start ES and Kibana                        |
 | --runner  | Only run tests                                  |
-| --grep    | Specify the spec files to run                   |
+| --grep    | Specify the specs to run                        |
+| --files   | Specify the files to run                        |
```

Comment:
Is this supposed to be `grep-files`?

Reply:
There's a `--files` alias but it's better to use `--grep-files` everywhere.